x/observer/types/chain_params.go

```diff
@@ -155,6 +155,34 @@ func ValidateChainParams(params *ChainParams) error {
 	return nil
 }
 
+// InboundConfirmationSafe returns the safe number of confirmation for inbound observation.
+func (cp ChainParams) InboundConfirmationSafe() uint64 {
+	return cp.ConfirmationParams.SafeInboundCount
+}
```

Comment:
just checking , should we have a fallback mechanism here to use Confirmation count in case Confimration params is not available. Wondering if in case we do a zetaclient only patch before the next release